### PR TITLE
Add in wercker to validate tests

### PR DESCRIPTION
**Pull request by bdangit**
- only test:quick is called
- eventually will build in rspec, functional testing 
  and a publish step

### CODE REVIEW

**Comment by bdangit** [comment]
@fishnix  this will be just for unit tests for now. 

We can merge this now or we can wait to have auto-version bump capabilities.  mid-term is to get kitchen tests automated as well. 

let me know what you think.

**Comment by bdangit** [comment]
@fishnix yay/nay. this will have capabilities to cut new releases.

File: wercker.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        - script:
+            name: get version from rake
+            code: |
+              export APP_VERSION=$(chef exec rake version)
+              echo $APP_VERSION
+        - github-create-release:
+            token: $GITHUB_TOKEN
+            tag: $APP_VERSION
+        # TODO: setup stove to push to supermarket
+
```
Comment (fishnix) [inline wercker.yml]: This indentation looks out of whack, 2 spaces?

**Comment by fishnix** [comment]
Thats my only comment, otherwise :+1:

File: wercker.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+box: ubuntu:15.04
+build:
```
Comment (fishnix) [inline wercker.yml]: Are we able to test at least one RHEL/CentoOS variant too?

**Comment by fishnix** [comment]
Sorry I had 2 comments :)

File: wercker.yml (hunk 1 above)
Comment (bdangit) [inline wercker.yml]: updated to 4 spaces...

**Comment by fishnix** [comment]
Sorry if I was confusing, I think it _should_ be 2 spaces.... like this?

```yaml

box: ubuntu:15.04
build:
  steps:
    - script:
        name: install wget
        code: |
          sudo apt-get update -y
          sudo apt-get install -y wget
    - poppen/chefdk:
        version: 0.10.0
    - script:
        name: test
        code: |
          chef exec "rake test:quick"
deploy:
  - install-packages:
      packages: wget curl
  - poppen/chefdk:
      version: 0.10.0
  - script:
      name: get version from rake
      code: |`
        export APP_VERSION=$(chef exec rake version)
        echo $APP_VERSION
  - github-create-release:
      token: $GITHUB_TOKEN
      tag: $APP_VERSION
  # TODO: setup stove to push to supermarket
```

File: wercker.yml (hunk 2 above)
Comment (bdangit) [inline wercker.yml]: this is just for building, its not intended to test the compatibility of the cookbook recipes.  Still working on how we can do test-kitchen.. I'm thinking of setting up test-kitchen-ec2 to do this.

**Comment by bdangit** [comment]
@fishnix review again.

File: wercker.yml (hunk 1 above)
Comment (bdangit) [inline wercker.yml]: updated to 2 spaces

**Comment by fishnix** [comment]
f4b83dac67da2ad4735707c1a4af62cc80181631 looks good :+1:

**Pull request merged**